index.html

```diff
@@ -4,11 +4,168 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Honky Tonk Music Fest</title>
+    <title>Home - Honky Tonk Music Fest</title>
+    <link
+      href="http://fonts.googleapis.com/css?family=Lato:400,700"
+      rel="stylesheet"
+      type="text/css"
+    />
     <link rel="stylesheet" href="style.css" />
+    <link
+      rel="icon"
+      href="assets/images/logo_honky_tonk.png"
+      type="image/x-icon"
+    />
   </head>
   <body>
-    <h1>Honky Tonk Music Festival</h1>
+    <nav>
+      <div class="hamburger">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+      <a href="index.html" id="logo"
+        ><img src="assets/images/header_logo.png"
+      /></a>
+      <div class="mobile__dropdown no__display">
+        <div class="cancel">
+          <div class="cancel__bar__1"></div>
+          <div class="cancel__bar__2"></div>
+        </div>
+        <ul>
+          <li>
+            <a href="index.html" id="page__indicator">Home</a>
+          </li>
+          <li>
+            <a href="about.html">About</a>
+          </li>
+          <li>
+            <button type="button" id="book__btn">Book Ticket(s)</button>
+          </li>
+        </ul>
+      </div>
+    </nav>
```

Comment:
- [ ] it would be better if you add the black sub nav bar as in the design in order to match the design